### PR TITLE
Use antitypical/Result instead of custom Result of Alamofire

### PR DESCRIPTION
**Pull request by davideme**

### CODE REVIEW

File: Alamofire+Result.podspec [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  s.social_media_url = 'http://twitter.com/goKarumi'
+  s.authors = { 'Alamofire Software Foundation' => 'info@alamofire.org',
+                'Karumi'                        => 'hello@karumi.com' }
+  s.source = { :git => 'https://github.com/Karumi/Alamofire.git', :tag => s.version }
+
+  s.ios.deployment_target = '8.0'
+  s.osx.deployment_target = '10.10'
+  s.watchos.deployment_target = '2.0'
+
+  s.dependency 'Result', '~> 0.6.0-beta.4'
```
Comment (pedrovgs) [inline Alamofire+Result.podspec]: Are we going to publish this version pointing to a beta version? I don't know the state of this library but If you think using this version could be fine it's ok for me.

**Comment by pedrovgs** [comment]
@davideme this PR LGTM but maybe we should update the README.md file with information about why this fork is going to use ``Result`` and not the original alamofire implementation.

File: Alamofire+Result.podspec (hunk 1 above)
Comment (pedrovgs) [inline Alamofire+Result.podspec]: If we are going to use this version we should create an issue to update the library version to 0.6.0 once be released.

Comment (davideme) [inline Alamofire+Result.podspec]: Yes we are going with the beta version, we are using the beta version for 2 months now in another project(ECI). Without any issues, and I will follow up theses repo to keep everything update.

Comment (pedrovgs) [inline Alamofire+Result.podspec]: ok!

**Comment by pedrovgs** [comment]
:+1:

**Pull request closed without merging**